doc/porting_guide.md

```diff
@@ -34,14 +34,20 @@ For more information please see the Xively C Client User Guide: `doc/user_guide.
 
 ## Supported Platforms
 
-The Xively Client has been deployed on many different devices already shipping in the IoT space running the following platform stacks:
+The Xively Client has been deployed on many different devices already shipping in the IoT space running the following platform stacks. The following is a catalog of our tested ports:
 
   - POSIX systems (OSX, Linux)
   - Microchip TCP SDK
-  - WMSDK
   - Texas Instruments CC3200
+  - Texas Instruments CC3220SF
   - STM3241G-EVAL
+  - STM32F4 Nucelo
```

Comment:
Nucelo->Nucleo.

Also, isn't that statement too broad? I thought some F4 Nucleo boards were not compatible with the WiFi module. I know it should be easy to port the code to a different F4 Nucleo, and most(all?) of the code should work right away, but I don't know if that's a "tested port" per se.

Same for `STM32F4 Nucleo Wifi`